src/table.tsx

```diff
@@ -51,11 +56,23 @@ export const Table = React.forwardRef(
             maxHeight,
             emptyCase,
             columnTextAlign,
+            rowVerticalAlign = 'middle',
             columnWidth,
             ...otherProps
         }: TableProps,
         ref: React.Ref<HTMLDivElement>
     ) => {
+        const getColumnTextAlign = (column: number) => {
+            const defaultTextAlign = 'left';
+            if (columnTextAlign) {
```

Comment:
Perhaps you can use a default parameter to simplify this to: `return Array.isArray(columnTextAlign) ? columnTextAlign[column] : columnTextAlign`